qcore/archive_structure.py

```diff
@@ -0,0 +1,20 @@
+"""
+Gives access to the folder structure of archived cybershake directories
+"""
+from pathlib import Path
+
+
+def get_fault_from_realisation(realisation: str):
+    """Gets the fault name from the realisation name"""
+    return realisation.rsplit("_REL", 1)[0]
```

Comment:
If this function is already in simulation structure, it should be imported instead of re-written. To make sure they do the same thing